### PR TITLE
Add watch access vectors.

### PR DESCRIPTION
**Pull request by pebenito**
Signed-off-by: Chris PeBenito <chpebeni@linux.microsoft.com>

### CODE REVIEW

**Comment by pebenito** [comment]
If I move the permissions to the common, is the breakage only for kernels that don't have the dynamic class/perm mapping? i.e. super old kernels?

**Comment by stephensmalley** [comment]
Yes, kernels < 2.6.33, < RHEL 6.  I guess refpolicy already broke compatibility with those kernels when it added the map permission to common file and socket.

**Comment by stephensmalley** [comment]
NB Dominick noted that changes to common/class file also broke old crond still shipping in some distros (Debian) that were still using the hardcoded FILE__ENTRYPOINT definition.  However, that too would have broken with the map permission change, so nothing new there, and I believe a patch was submited for cron.

**Comment by pebenito** [comment]
Thanks Stephen for jogging my memory.

File: INSTALL [hunk 1]
```diff
@@ -1,3 +1,6 @@
+Reference Policy has the following runtime requirements:
+	* Linux kernel >= 2.6.34
+
```
Comment (stephensmalley) [inline INSTALL]: 2.6.33

File: policy/flask/access_vectors [hunk 2]
```diff
@@ -28,6 +28,9 @@ common file
 	swapon
 	quotaon
 	mounton
+	open
+	audit_access
+	execmod
```
Comment (stephensmalley) [inline policy/flask/access_vectors]: kernel and fedora have:
audit_access
open
execmod

**Pull request merged**